pkg/storage/replica_proposal.go

```diff
@@ -600,22 +609,6 @@ func (r *Replica) handleLocalEvalResult(
 	// Non-state updates and actions.
 	// ======================
 
-	if originReplica.StoreID == r.store.StoreID() {
```

Comment:
Yes, suggestion was to make sure it's detached here with an assertion and a comment.